src/main/java/uk/gov/hmcts/reform/civil/helpers/judgmentsonline/JudgmentByAdmissionOnlineMapper.java

```diff
@@ -33,16 +38,21 @@ public class JudgmentByAdmissionOnlineMapper extends JudgmentOnlineMapper {
 
     boolean isNonDivergent = false;
     private final RoboticsAddressMapper addressMapper;
+    private final InterestCalculator interestCalculator;
 
     @Override
     public JudgmentDetails addUpdateActiveJudgment(CaseData caseData) {
+        var interest = interestCalculator.calculateInterest(caseData);
+        caseData.setTotalInterest(interest);
```

Comment:
Why we need to this calculation again? Is it possible to use these method/field which already doing this calculation in 
![image](https://github.com/user-attachments/assets/03e2c5c1-9058-4b84-a304-936ee0a8b6eb)